cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java

```diff
@@ -423,6 +424,33 @@ public Long getResult() {
     return processor.getResult();
   }
 
+  /**
+   * Constructs a table name for stream consumer state stores within a given namespace.
+   * @param namespace the namespace for which the table is for.
+   * @return constructed table name
+   */
+  public static String constructStateStoreTableName(Id.Namespace namespace) {
+    return String.format("cdap.%s.%s.state.store", namespace.getId(), QueueConstants.STREAM_TABLE_PREFIX);
```

Comment:
should add the hack we spoke about for default namespace here, no? Or is that coming later?